Ideology/Keyed/GameplayCommands.xml

```diff
@@ -26,5 +26,9 @@
   <CommandNeuralSuperchargerAllowGuestsDescription>Разрешить гостям пользоваться этим нейронным стимулятором.</CommandNeuralSuperchargerAllowGuestsDescription>
 
   <!-- EN: Select connected tree -->
-  <CommandSelectConnectedTree>Выбрать связанное дерево</CommandSelectConnectedTree>
-</LanguageData>
+  <CommandSelectConnectedTree>Выбрать связанное древо</CommandSelectConnectedTree>
+  <!-- EN: Create healing pod -->
+  <CommandHealInHealingPod>Создать исцеляющий кокон</CommandHealInHealingPod>
+  <!-- EN: If this creature is injured, it will return to its connected Gauranlen tree and enter a woody healing pod. After {DAYS} days, the dryad will emerge with its wounds healed. -->
+  <CommandHealInHealingPodDesc>При ранении дриада вернётся к своему древу Гауранлен и войдёт в древесный исцеляющий кокон. Через {DAYS_numCase ? дня : дней : дней} в коконе её раны залечатся, и она покинет его.</CommandHealInHealingPodDesc>
```

Comment:
```suggestion
  <CommandHealInHealingPod>Создать исцеляющий кокон</CommandHealInHealingPod>
  <!-- EN: If this creature is injured, it will return to its connected Gauranlen tree and enter a woody healing pod. After {DAYS} days, the dryad will emerge with its wounds healed. -->
  <CommandHealInHealingPodDesc>При ранении дриада вернётся к своему древу Гауранлен и войдёт в древесный исцеляющий кокон. Через {DAYS_numCase ? дня : дней : дней} в коконе её раны залечатся, и она покинет его.</CommandHealInHealingPodDesc>
```

Я помню, что предлагал замену на регенеративный, но не помню, приняли ли мы её в итоге.